.github/workflows/ci.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         git remote add upstream https://github.com/pandas-dev/pandas.git
         git fetch upstream
         asv machine --yes
-        asv dev | sed "/failed$/ s/^/##[error]/" | tee benchmarks.log
+        asv run --quick --show-stderr --python=same --set-commit-hash HEAD | sed "/failed$/ s/^/##[error]/" | tee benchmarks.log
```

Comment:
This change should be unnecessary. I think the asv docs say that ``asv dev`` is
equivalent to ``asv run --quick --show-stderr --python=same``

Reply:
Yes, but in practice it's not exactly like that. Here specifically I needed to specify `--set-commit-hash` to have it save results (that is documented), and this keyword is not passed through by `asv dev`.

Reply:
Thanks for the clarification.